src/lib/pages/backoffice/Document/DocumentForms/DocumentEditor/DocumentForm/components/Imprint.js

```diff
@@ -17,7 +18,7 @@ export class Imprint extends Component {
                 fieldPath="imprint.publisher"
                 optimized
               />
-              <StringField
+              <DateInputField
```

Comment:
are we sure about this change? Backend is expecting a string, and we might want to only input a year, not a full date, same below.

Reply:
datepicker returns the string, so the type should be OK
but good point about the year only, @KonstantinaStoikou can you remove this change?